wire-runtime/src/main/java/com/squareup/wire/ProtoAdapter.java

```diff
@@ -66,6 +67,12 @@ public ProtoAdapter(FieldEncoding fieldEncoding, Class<?> javaType) {
     return new RuntimeEnumAdapter<>(type);
   }
 
+  /** Creates a new proto adapter for a map using {@code keyAdapter} and {@code valueAdapter}. */
+  public static <K, V> ProtoAdapter<Map<K, V>> createMapAdapter(ProtoAdapter<K> keyAdapter,
```

Comment:
weirdly inconsistent for the line above `newEnumAdapter` and this `createMapAdapter` . . . maybe `newMapAdapter` ?